app/src/app/components/specmath-modal/modal.component.html

```diff
@@ -51,7 +51,7 @@ <h3>
         ></app-spec-files-upload>
       </mat-step>
       <mat-step>
-        <app-confirm-operation
+        <app-confirm-operation *ngIf="validFiles"
```

Comment:
Perhaps it would make sense to add a test of this behavior?

Reply:
Updated 👍